rero_ils/modules/loans/cli.py

```diff
@@ -395,13 +401,13 @@ def create_loan(barcode, transaction_type, loanable_items, verbose=False,
                 notification_pids.extend(
                     notif['pid'] for notif in notifications)
         Dispatcher.dispatch_notifications(notification_pids, verbose=verbose)
-        return item['barcode']
+        return item['barcode'], ''
     except Exception as err:
         if verbose:
             click.secho(f'\tException loan {transaction_type}:{err}', fg='red')
         if debug:
             traceback.print_exc()
-        return None, []
+        return item['barcode'], err
```

Comment:
strange to return an exeception as second argument of the tuple response.... maybe better to return 'None' ?

Reply:
Changed it.